_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -5125,10 +5128,36 @@ mfxStatus MfxHwH264Encode::CheckAndFixMVCSeqDesc(mfxExtMVCSeqDesc * mvcSeqDesc,
 void MfxHwH264Encode::InheritDefaultValues(
     MfxVideoParam const & parInit,
     MfxVideoParam &       parReset,
+    MFX_ENCODE_CAPS const & hwCaps,
```

Comment:
Also you already use necessary struct :)

Reply:
@AntonGrishin : clarify, please, I did not get the point.